HealthcareSystem/GraphicalEditor/Service/Interface/IMapObjectServices.cs

```diff
@@ -0,0 +1,14 @@
+﻿using GraphicalEditor.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicalEditor.Services.Interface
+{
+    public interface IMapObjectServices
+    {
+        bool Update(MapObject mapObject);
+    }
```

Comment:
Again, what we are Updating, and why this has bool return value?

Reply:
to check if it will update object succesfully

Reply:
Using bool for that should generally be avoided. Use return value to return actual results of an operation, and exceptions for error handling.

Reply:
In our infrastructure, we write an exception if something goes wrong. Try to avoid bool return type for these methods.

Reply:
done